package.json

```diff
@@ -35,6 +35,15 @@
     "vuetify": "^3.0.1"
   },
   "dependencies": {
+    "@fullcalendar/core": "^6.1.4",
```

Comment:
OK, zdej pa VEM, da je treba za example app narest ločen package.json. Dynamicforms sami ga ne rabijo, razen če rečemo, da je calendar komponenta nek "field" v knjižnjici.

Alternativa: move this to DevDependencies?

Reply:
Odprl task, da se pogleda, kako imajo drugi repoti to reseno